diff --git i/eatmydatatest.c w/eatmydatatest.c
index dc131d9..c5f6fd3 100644
--- i/eatmydatatest.c
+++ w/eatmydatatest.c
@@ -19,6 +19,7 @@
 #include <errno.h>
 #include <sys/mman.h>
 #include <unistd.h>
+#include <sys/stat.h>
 
 int failed_tests = 0;
 
@@ -37,7 +38,7 @@ int test_open_flag(int flag)
     int fd, res;
 
     res = -1;
-    fd = open(DATAFILENAME, O_CREAT|O_TRUNC|O_WRONLY|flag);
+    fd = open(DATAFILENAME, O_CREAT|O_TRUNC|O_WRONLY|flag, S_IRWXU);
     if (fd != -1) {
         res = fcntl(fd, F_GETFL);
     }
@@ -61,7 +62,7 @@ int main(int argc, char** argv)
     TESTCASE(test_open_flag(O_DSYNC), 0, "open(O_DSYNC) flag was not filtered out");
 
     // Open and close FD in order to make sure its invalid.
-    fd = open(DATAFILENAME, O_CREAT|O_TRUNC|O_WRONLY);
+    fd = open(DATAFILENAME, O_CREAT|O_TRUNC|O_WRONLY, S_IRWXU);
     if (fd == -1)
         perror("unable to open " DATAFILENAME);
     if (close(fd) == -1)
