From 4c7ce5a9949947f41be83098abea4bf5f5d71862 Mon Sep 17 00:00:00 2001
From: Brett Parker <iDunno@sommitrealweird.co.uk>
Date: Mon, 15 Mar 2010 00:21:31 +0000
Subject: [PATCH] Make work with Net::Twitter::Lite


---
 twirssi.pl |   80 ++++++------------------------------------------------------
 1 files changed, 8 insertions(+), 72 deletions(-)

diff --git a/twirssi.pl b/twirssi.pl
index 7147981..45faf6f 100644
--- a/twirssi.pl
+++ b/twirssi.pl
@@ -8,7 +8,7 @@ use LWP::Simple;
 use Data::Dumper;
 use Encode;
 use POSIX qw/:sys_wait_h/;
-use Net::Twitter qw/3.05/;
+use Net::Twitter::Lite
 $Data::Dumper::Indent = 1;
 
 use vars qw($VERSION %IRSSI);
@@ -509,74 +509,10 @@ sub cmd_login {
     }
     $defservice = $service = ucfirst lc $service;
 
-    if ( $service eq 'Twitter'
-        and Irssi::settings_get_bool("twirssi_use_oauth") )
-    {
-        print "Attempting OAuth for $user\@$service" if &debug;
-        eval {
-            if ( $service eq 'Identica' )
-            {
-                $twit = Net::Twitter->new(
-                    identica => 1,
-                    traits   => [ 'API::REST', 'API::Search' ],
-                    source   => "twirssi",
-                    ssl      => !Irssi::settings_get_bool("twirssi_avoid_ssl"),
-                );
-            } else {
-                $twit = Net::Twitter->new(
-                    traits => [ 'API::REST', 'OAuth', 'API::Search' ],
-                    consumer_key => 'BZVAvBma4GxdiRwXIvbnw',
-                    consumer_secret =>
-                      '0T5kahwLyb34vciGZsgkA9lsjtGCQ05vxVE2APXM',
-                    source => "twirssi",
-                    ssl    => !Irssi::settings_get_bool("twirssi_avoid_ssl"),
-                );
-            }
-        };
-
-        if ($twit) {
-            if ( open( OAUTH, Irssi::settings_get_str("twirssi_oauth_store") ) )
-            {
-                while (<OAUTH>) {
-                    chomp;
-                    next unless m/$user\@$service (\S+) (\S+)/i;
-                    print "Trying cached oauth creds for $user\@$service"
-                      if &debug;
-                    $twit->access_token($1);
-                    $twit->access_token_secret($2);
-                    last;
-                }
-                close OAUTH;
-            }
-
-            unless ( $twit->authorized ) {
-                my $url;
-                eval { $url = $twit->get_authorization_url; };
-
-                if ($@) {
-                    &notice( "ERROR: Failed to get OAuth authorization_url. " .
-                             "Try again later.");
-                    return;
-                }
-                &notice( "Twirssi not autorized to access $service for $user.",
-                         "Please authorize at the following url, then enter the pin",
-                         "supplied with /twirssi_oauth $user\@$service <pin>",
-                         $url
-                );
-
-                $oauth{pending}{"$user\@$service"} = $twit;
-                return;
-            }
-        }
-    } else {
-        $twit = Net::Twitter->new(
-            $service eq 'Identica' ? ( identica => 1 ) : (),
-            username => $user,
-            password => $pass,
-            source   => "twirssi",
-            ssl      => Irssi::settings_get_bool("twirssi_avoid_ssl") ? 0 : 1,
-        );
-    }
+    $twit = Net::Twitter::Lite->new(
+        username => $user,
+        password => $pass,
+    );
 
     unless ($twit) {
         &notice("Failed to create object!  Aborting.");
@@ -1808,7 +1744,7 @@ sub get_text {
         if ( exists $tweet->{retweeted_status} ) {
             $text = "RT \@$tweet->{retweeted_status}{user}{screen_name}: "
               . "$tweet->{retweeted_status}{text}";
-        } elsif ( $object->isa('Net::Twitter') ) {
+        } elsif ( $object->isa('Net::Twitter::Lite') ) {
             $text .= " -- http://twitter.com/$tweet->{user}{screen_name}"
               . "/status/$tweet->{id}";
         }
@@ -1931,8 +1867,8 @@ if ($window) {
             &notice(
                 "Twirssi v$VERSION; "
                   . (
-                    $Net::Twitter::VERSION
-                    ? "Net::Twitter v$Net::Twitter::VERSION. "
+                    $Net::Twitter::Lite::VERSION
+                    ? "Net::Twitter v$Net::Twitter::Lite::VERSION. "
                     : ""
                   )
                   . (
-- 
1.6.4.4

