From 7df013a70c80fc821c6a71ac01803b93add59076 Mon Sep 17 00:00:00 2001
From: James Rowe <jnrowe@ukfsn.org>
Date: Tue, 3 Feb 2009 11:44:41 +0000
Subject: [PATCH] Don't create config file unless we're using arch.

---
 libbe/arch.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libbe/arch.py b/libbe/arch.py
index fd953a4..e8be4ae 100644
--- a/libbe/arch.py
+++ b/libbe/arch.py
@@ -28,7 +28,6 @@ from rcs import RCS, RCStestCase, CommandError
 client = config.get_val("arch_client")
 if client is None:
     client = "tla"
-    config.set_val("arch_client", client)
 
 def new():
     return Arch()
@@ -49,6 +48,7 @@ class Arch(RCS):
     def _rcs_detect(self, path):
         """Detect whether a directory is revision-controlled using Arch"""
         if self._u_search_parent_directories(path, "{arch}") != None :
+            config.set_val("arch_client", client)
             return True
         return False
     def _rcs_init(self, path):
-- 
1.6.0.6

