From: James Rowe <jnrowe@gmail.com>
Subject: Use 2to3 for Python 3 compatibility.
Status: LOCAL

diff --git i/setup.py w/setup.py
index 24368e6..57ad853 100644
--- i/setup.py
+++ w/setup.py
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+from setuptools import setup
 
 setup(name='schematics',
       license='BSD',
@@ -9,4 +9,5 @@ setup(name='schematics',
       author='James Dennis',
       author_email='jdennis@gmail.com',
       url='http://github.com/j2labs/schematics',
-      packages=['schematics', 'schematics.types'])
+      packages=['schematics', 'schematics.types'],
+      use_2to3 = True)
