From: James Rowe <jnrowe@gmail.com>
Subject: Unbundle libraries, to fight off the QA nazis^wfolks
Status: LOCAL

diff --git a/requests.egg-info/SOURCES.txt b/requests.egg-info/SOURCES.txt
index 65c3b1d..573f0ec 100644
--- a/requests.egg-info/SOURCES.txt
+++ b/requests.egg-info/SOURCES.txt
@@ -20,8 +20,5 @@ requests/utils.py
 requests.egg-info/PKG-INFO
 requests.egg-info/SOURCES.txt
 requests.egg-info/dependency_links.txt
-requests.egg-info/top_level.txt
-requests/packages/__init__.py
-requests/packages/poster/__init__.py
-requests/packages/poster/encode.py
-requests/packages/poster/streaminghttp.py
\ No newline at end of file
+requests.egg-info/requires.txt
+requests.egg-info/top_level.txt
\ No newline at end of file
diff --git a/requests.egg-info/requires.txt b/requests.egg-info/requires.txt
new file mode 100644
index 0000000..0b356ef
--- /dev/null
+++ b/requests.egg-info/requires.txt
@@ -0,0 +1 @@
+poster
\ No newline at end of file
diff --git a/requests/models.py b/requests/models.py
index 2d7fc8f..b907a47 100644
--- a/requests/models.py
+++ b/requests/models.py
@@ -15,11 +15,12 @@ from urllib2 import HTTPError
 from urlparse import urlparse, urlunparse, urljoin
 from datetime import datetime
 
+from poster.encode import multipart_encode
+from poster.streaminghttp import register_openers, get_handlers
+
 from .config import settings
 from .monkeys import Request as _Request, HTTPBasicAuthHandler, HTTPForcedBasicAuthHandler, HTTPDigestAuthHandler, HTTPRedirectHandler
 from .structures import CaseInsensitiveDict
-from .packages.poster.encode import multipart_encode
-from .packages.poster.streaminghttp import register_openers, get_handlers
 from .utils import dict_from_cookiejar
 from .exceptions import RequestException, AuthenticationError, Timeout, URLRequired, InvalidMethod, TooManyRedirects
 from .status_codes import codes
diff --git a/setup.py b/setup.py
index 6b95c31..5d5789f 100755
--- a/setup.py
+++ b/setup.py
@@ -20,7 +20,7 @@ if sys.argv[-1] == "test":
     os.system("python test_requests.py")
     sys.exit()
 
-required = []
+required = ['poster']
 
 if sys.version_info[:2] < (2,6):
     required.append('simplejson')
@@ -36,8 +36,6 @@ setup(
     url='http://python-requests.org',
     packages= [
         'requests',
-        'requests.packages',
-        'requests.packages.poster'
     ],
     install_requires=required,
     license='ISC',
