From: James Rowe <jnrowe@gmail.com>
Subject: Install to /usr, DESTDIR support, FHS compliance
Status: LOCAL

diff --git i/Makefile w/Makefile
index 6f82544..78c7f16 100644
--- i/Makefile
+++ w/Makefile
@@ -26,7 +26,7 @@
 # those of the authors and should not be interpreted as representing official
 # policies, either expressed or implied, of Vincent Driessen.
 #
-prefix=/usr/local
+prefix=/usr
 
 # files that need mode 755
 EXEC_FILES=git-flow
@@ -47,9 +47,11 @@ all:
 
 install:
 	@test -f gitflow-shFlags || (echo "Run 'git submodule init && git submodule update' first." ; exit 1 )
-	install -d -m 0755 $(prefix)/bin
-	install -m 0755 $(EXEC_FILES) $(prefix)/bin
-	install -m 0644 $(SCRIPT_FILES) $(prefix)/bin
+	install -d $(DESTDIR)$(prefix)/bin
+	install -d $(DESTDIR)$(prefix)/share/gitflow
+	install $(EXEC_FILES) $(DESTDIR)$(prefix)/bin
+	install -m 0644 $(SCRIPT_FILES) $(DESTDIR)$(prefix)/share/gitflow
+
 
 uninstall:
 	test -d $(prefix)/bin && \
diff --git i/git-flow w/git-flow
index e95d4ad..119cd4d 100755
--- i/git-flow
+++ w/git-flow
@@ -42,7 +42,7 @@ if [ "$DEBUG" = "yes" ]; then
 	set -x
 fi
 
-export GITFLOW_DIR=$(dirname "$0")
+export GITFLOW_DIR=/usr/share/gitflow
 
 usage() {
 	echo "usage: git flow <subcommand>"
