diff --git i/main.cpp w/main.cpp
index 386bb3a..c5f9c95 100644
--- i/main.cpp
+++ w/main.cpp
@@ -16,6 +16,8 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
  */
 
+#include <sys/stat.h>
+
 #include "file.h"
 #include "game.h"
 #include "systemstub.h"
@@ -23,10 +25,10 @@
 static const char *USAGE =
 	"REminiscence - Flashback Interpreter\n"
 	"Usage: rs [OPTIONS]...\n"
-	"  --datapath=PATH   Path to data files (default 'DATA')\n"
-	"  --savepath=PATH   Path to save files (default '.')";
+	"  --datapath=PATH   Path to data files (default 'GAMES_DATADIR')\n"
+	"  --savepath=PATH   Path to save files (default '~/.reminiscence')";
 
-static bool parseOption(const char *arg, const char *longCmd, const char **opt) {
+static bool parseOption(char *arg, const char *longCmd, char **opt) {
 	bool handled = false;
 	if (arg[0] == '-' && arg[1] == '-') {
 		if (strncmp(arg + 2, longCmd, strlen(longCmd)) == 0) {
@@ -59,8 +61,17 @@ static Version detectVersion(const char *dataPath) {
 
 #undef main
 int main(int argc, char *argv[]) {
-	const char *dataPath = "DATA";
-	const char *savePath = ".";
+	char *dataPath = (char *)"GAMES_DATADIR";
+	char *savePath = (char *)malloc(256);
+	struct stat sb;
+
+	snprintf(savePath, 255, "%s/.reminiscence", getenv("HOME"));
+	if (stat(savePath, &sb)) {
+		if (mkdir(savePath, 0755) == -1) {
+			error("Failed to create save directory(%s)", savePath);
+		}
+	}
+
 	for (int i = 1; i < argc; ++i) {
 		bool opt = false;
 		if (strlen(argv[i]) >= 2) {
