diff --git i/Makefile w/Makefile
index 2840eb4..545f7e4 100644
--- i/Makefile
+++ w/Makefile
@@ -4,7 +4,7 @@ VALA_CFLAGS=-g --pkg gtk+-2.0 --pkg gio-unix-2.0 --pkg gio-2.0 --pkg pango --pkg
 all: notmuch_gtk
 
 notmuch_gtk: $(VALA_SRC) Makefile
-	valac $(VALA_CFLAGS) -o notmuch_gtk $(VALA_SRC)
+	valac-0.10 $(VALA_CFLAGS) -o notmuch_gtk $(VALA_SRC)
 
 clean:
 	-rm -f notmuch_gtk $(wildcard src/*.c)
diff --git i/src/exec.vala w/src/exec.vala
index adb0157..b67d903 100644
--- i/src/exec.vala
+++ w/src/exec.vala
@@ -114,7 +114,7 @@ namespace NotMuch.Exec {
 		int count = parts.length;
 		
 		string[] argv = new string[3+count];
-		argv[0] = "/usr/local/bin/notmuch";
+		argv[0] = "/usr/bin/notmuch";
 		argv[1] = "search";
 		
 		for (int i = 0; i < count; i++) {
@@ -145,7 +145,7 @@ namespace NotMuch.Exec {
 
 		int argv_idx = 0;;
 		string[] argv = new string[4 + parts_count + add_tags.length + remove_tags.length];
-		argv[argv_idx++] = "/usr/local/bin/notmuch";
+		argv[argv_idx++] = "/usr/bin/notmuch";
 		argv[argv_idx++] = "tag";
 		add_array(add_tags, '+', ref argv, ref argv_idx);
 		add_array(remove_tags, '-', ref argv, ref argv_idx);
@@ -158,7 +158,7 @@ namespace NotMuch.Exec {
 
 	public Executor thread_read(string query) {
 		string[] argv = new string[4];
-		argv[0] = "/usr/local/bin/notmuch";
+		argv[0] = "/usr/bin/notmuch";
 		argv[1] = "show";
 		argv[2] = query;
 		argv[3] = null;
diff --git i/src/main.vala w/src/main.vala
index 0088400..dd017e6 100644
--- i/src/main.vala
+++ w/src/main.vala
@@ -8,7 +8,7 @@ public static int main(string[] args)
 
 	Global.builder = new Gtk.Builder();
 	try {
-		Global.builder.add_from_file("glade/notmuch-gtk.glade");
+		Global.builder.add_from_file("/usr/share/notmuch-gtk/notmuch-gtk.glade");
 		Global.builder.connect_signals(null);
 	} catch (GLib.Error e) {
 		error("Error reading glade file: %s", e.message);
