From: James Rowe <jnrowe@gmail.com>
Subject: argparse only required with Python 2.6
Status: LOCAL

diff --git i/setup.py w/setup.py
index 5b3307d..977b905 100644
--- i/setup.py
+++ w/setup.py
@@ -1,6 +1,14 @@
 from os.path import join, dirname
+from sys import version_info
+
 from setuptools import setup
 
+if version_info[:2] == (2, 6):
+    requires = ['argparse', ]
+else:
+    requires = []
+
+
 setup(name='aaargh',
       version='0.4',  # XXX: Keep version number in sync with setup.py
       description='An astonishingly awesome application argument helper',
@@ -9,7 +17,7 @@ setup(name='aaargh',
       author_email='uws@xs4all.nl',
       url='https://github.com/wbolster/aaargh',
       py_modules=['aaargh'],
-      install_requires=['argparse'],
+      install_requires=requires,
       license='BSD',
       classifiers=(
           'Development Status :: 4 - Beta',
