diff --git i/Makefile.in w/Makefile.in
index 9cecfe2..643b6a3 100644
--- i/Makefile.in
+++ w/Makefile.in
@@ -66,13 +66,13 @@ termdebug: $(OBJECTS)
 	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(OBJECTS) $(LDLIBS) $(CONFIGLIBS)
 
 install: all
-	$(INSTALL) -d "$(bindir)"
-	$(INSTALL) termdebug "$(bindir)"
-	cd "$(bindir)" && ln -s termdebug tdrecord
-	cd "$(bindir)" && ln -s termdebug tdreplay
-	cd "$(bindir)" && ln -s termdebug tdrerecord
-	cd "$(bindir)" && ln -s termdebug tdview
-	$(INSTALL) -d "$(mandir)/man1"
-	$(INSTALL) -m 644 man/*.1 "$(mandir)/man1"
-	$(INSTALL) -d "$(docdir)"
-	$(INSTALL) -m 644 README COPYING Changelog "$(docdir)"
+	$(INSTALL) -d "$(DESTDIR)$(bindir)"
+	$(INSTALL) termdebug "$(DESTDIR)$(bindir)"
+	cd "$(DESTDIR)$(bindir)" && ln -s termdebug tdrecord
+	cd "$(DESTDIR)$(bindir)" && ln -s termdebug tdreplay
+	cd "$(DESTDIR)$(bindir)" && ln -s termdebug tdrerecord
+	cd "$(DESTDIR)$(bindir)" && ln -s termdebug tdview
+	$(INSTALL) -d "$(DESTDIR)$(mandir)/man1"
+	$(INSTALL) -m 644 man/*.1 "$(DESTDIR)$(mandir)/man1"
+	$(INSTALL) -d "$(DESTDIR)$(docdir)"
+	$(INSTALL) -m 644 README COPYING Changelog "$(DESTDIR)$(docdir)"
