From: James Rowe <jnrowe@gmail.com>
Subject: Fix .desktop file's validation errors
Status: LOCAL

diff --git i/gpxviewer.desktop.in w/gpxviewer.desktop.in
index 7f402e2..df829c6 100644
--- i/gpxviewer.desktop.in
+++ w/gpxviewer.desktop.in
@@ -4,5 +4,5 @@ Type=Application
 _Comment=GPS Trace Viewer
 Exec=gpxviewer
 Icon=gpxviewer
-Categories=Network
-MimeType=application/xml
+Categories=Network;
+MimeType=application/xml;
