From: James Rowe <jnrowe@gmail.com>
Subject: Fix encoding bug
Status: LOCAL

diff --git i/setup.py w/setup.py
index b201a7b..d5fc49a 100644
--- i/setup.py
+++ w/setup.py
@@ -9,7 +9,7 @@ setup(
     name = "html",
     version = __version__,
     description = "simple, elegant HTML, XHTML and XML generation",
-    long_description = __doc__.decode('utf8'),
+    long_description = __doc__,
     author = "Richard Jones",
     author_email = "rjones@ekit-inc.com",
     py_modules = ['html'],
