From 09df3169a1e48e54496515c9da6764d08cb7f1af Mon Sep 17 00:00:00 2001
From: James Rowe <jnrowe@gmail.com>
Date: Thu, 23 Sep 2010 09:06:33 +0100
Subject: [PATCH 1/2] Use system sqlite.


Signed-off-by: James Rowe <jnrowe@gmail.com>
---
 Makefile.am         |   11 +++++------
 headers/Makefile.am |    2 --
 2 files changed, 5 insertions(+), 8 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 0de23ba..bc9bbd7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -7,8 +7,7 @@ INCLUDES += @PROJ_INCLUDE@ @GEOS_INCLUDE@
 
 lib_LTLIBRARIES = libspatialite.la
 
-libspatialite_la_SOURCES = spatialite.c \
-	sqlite3.c
+libspatialite_la_SOURCES = spatialite.c
 
 if MINGW
 libspatialite_la_LDFLAGS = -version-info 2:3:1 -no-undefined
@@ -16,7 +15,7 @@ libspatialite_la_LIBADD = \
 	@GEOS_LIB_CLA@ \
 	@GEOS_LIB_LA@ \
 	@PROJ_LIB_LA@ \
-	-liconv -lm -lstdc++
+	-liconv -lm -lstdc++ -lsqlite3
 else
 if MACOSX
 libspatialite_la_LDFLAGS = -version-info 2:3:1
@@ -24,18 +23,18 @@ libspatialite_la_LIBADD = \
 	@GEOS_LIB_CLA@ \
 	@GEOS_LIB_LA@ \
 	@PROJ_LIB_LA@ \
-	-liconv -lm -lstdc++ -lpthread -ldl
+	-liconv -lm -lstdc++ -lpthread -ldl -lsqlite3
 else
 libspatialite_la_LDFLAGS = -version-info 2:3:1
 libspatialite_la_LIBADD = \
 	@GEOS_LIB_CLA@ \
 	@GEOS_LIB_LA@ \
 	@PROJ_LIB_LA@ \
-	-lm -lstdc++ -lpthread -ldl
+	-lm -lstdc++ -lpthread -ldl -lsqlite3
 endif
 endif
 
-AUTOMAKE_OPTIONS = dist-zip
+AUTOMAKE_OPTIONS = foreign dist-zip
 
 EXTRA_DIST = makefile.vc nmake.opt libspatialite.def
 
diff --git a/headers/Makefile.am b/headers/Makefile.am
index 757a002..ba75d58 100644
--- a/headers/Makefile.am
+++ b/headers/Makefile.am
@@ -4,6 +4,4 @@ nobase_include_HEADERS = \
 	spatialite/gaiaexif.h \
 	spatialite/gaiaaux.h \
 	spatialite/gaiageo.h \
-	spatialite/sqlite3.h \
-	spatialite/sqlite3ext.h \
 	spatialite/spatialite.h
-- 
1.7.2.2

