From da3934983abd1005341181cd0feba3af183d41d1 Mon Sep 17 00:00:00 2001
From: James Rowe <jnrowe@gmail.com>
Date: Thu, 23 Sep 2010 09:08:04 +0100
Subject: [PATCH 2/2] Use system libs directly, skip libtool archives.


Signed-off-by: James Rowe <jnrowe@gmail.com>
---
 Makefile.am  |   18 +++++++++---------
 configure.ac |   30 +++++++++++++++---------------
 2 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index bc9bbd7..149ccd5 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -12,24 +12,24 @@ libspatialite_la_SOURCES = spatialite.c
 if MINGW
 libspatialite_la_LDFLAGS = -version-info 2:3:1 -no-undefined
 libspatialite_la_LIBADD = \
-	@GEOS_LIB_CLA@ \
-	@GEOS_LIB_LA@ \
-	@PROJ_LIB_LA@ \
+	@GEOS_LIB_C@ \
+	@GEOS_LIB@ \
+	@PROJ_LIB@ \
 	-liconv -lm -lstdc++ -lsqlite3
 else
 if MACOSX
 libspatialite_la_LDFLAGS = -version-info 2:3:1
 libspatialite_la_LIBADD = \
-	@GEOS_LIB_CLA@ \
-	@GEOS_LIB_LA@ \
-	@PROJ_LIB_LA@ \
+	@GEOS_LIB_C@ \
+	@GEOS_LIB@ \
+	@PROJ_LIB@ \
 	-liconv -lm -lstdc++ -lpthread -ldl -lsqlite3
 else
 libspatialite_la_LDFLAGS = -version-info 2:3:1
 libspatialite_la_LIBADD = \
-	@GEOS_LIB_CLA@ \
-	@GEOS_LIB_LA@ \
-	@PROJ_LIB_LA@ \
+	@GEOS_LIB_C@ \
+	@GEOS_LIB@ \
+	@PROJ_LIB@ \
 	-lm -lstdc++ -lpthread -ldl -lsqlite3
 endif
 endif
diff --git a/configure.ac b/configure.ac
index 59f76c5..f27cccd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -88,11 +88,11 @@ AC_ARG_WITH(proj-lib,
   [  --with-proj-lib=DIR     search for PROJ.4 lib in DIR],
   proj_lib="$withval", proj_lib="")
 if test "$proj_lib" == "" ; then
-  PROJ_LIB_LA="/usr/local/lib/libproj.la"
+  PROJ_LIB="-lproj"
 else
-  PROJ_LIB_LA="$proj_lib/libproj.la"
+  PROJ_LIB="-L$proj_lib -lproj"
 fi
-AC_SUBST(PROJ_LIB_LA)
+AC_SUBST(PROJ_LIB)
 #-----------------------------------------------------------------------
 
 #-----------------------------------------------------------------------
@@ -105,8 +105,8 @@ if test x"$enable_proj" != "xno"; then
   OMIT_PROJ_FLAGS=
 else
   OMIT_PROJ_FLAGS=-DOMIT_PROJ
-  PROJ_LIB_LA=
-  AC_SUBST(PROJ_LIB_LA)
+  PROJ_LIB=
+  AC_SUBST(PROJ_LIB)
 fi
 AC_SUBST(OMIT_PROJ_FLAGS)
 #-----------------------------------------------------------------------
@@ -132,14 +132,14 @@ AC_ARG_WITH(geos-lib,
   [  --with-geos-lib=DIR     search for GEOS lib in DIR],
   geos_lib="$withval", geos_lib="")
 if test "$geos_lib" == "" ; then
-  GEOS_LIB_CLA="/usr/local/lib/libgeos_c.la"
-  GEOS_LIB_LA="/usr/local/lib/libgeos.la"
+  GEOS_LIB_C="lgeos_c"
+  GEOS_LIB="-lgeos"
 else
-  GEOS_LIB_CLA="$geos_lib/libgeos_c.la"
-  GEOS_LIB_LA="$geos_lib/libgeos.la"
+  GEOS_LIB_C="-L$geos_lib -lgeos_c"
+  GEOS_LIB="-L$geos_lib -lgeos"
 fi
-AC_SUBST(GEOS_LIB_CLA)
-AC_SUBST(GEOS_LIB_LA)
+AC_SUBST(GEOS_LIB_C)
+AC_SUBST(GEOS_LIB)
 #-----------------------------------------------------------------------
 
 #-----------------------------------------------------------------------
@@ -152,10 +152,10 @@ if test x"$enable_geos" != "xno"; then
   OMIT_GEOS_FLAGS=
 else
   OMIT_GEOS_FLAGS=-DOMIT_GEOS
-  GEOS_LIB_CLA=
-  GEOS_LIB_LA=
-  AC_SUBST(GEOS_LIB_CLA)
-  AC_SUBST(GEOS_LIB_LA)
+  GEOS_LIB_C=
+  GEOS_LIB=
+  AC_SUBST(GEOS_LIB_C)
+  AC_SUBST(GEOS_LIB)
 fi
 AC_SUBST(OMIT_GEOS_FLAGS)
 #-----------------------------------------------------------------------
-- 
1.7.2.2

