From: James Rowe <jnrowe@gmail.com>
Subject: Install as git-serve, for use as 'git serve'
Status: LOCAL

diff --git i/setup.py w/setup.py
index ec5f573..728baa5 100644
--- i/setup.py
+++ w/setup.py
@@ -48,7 +48,7 @@ setup(
     packages=find_packages('src'),
     package_dir={'':'src'},
     package_data={'': ['media/*.*', '*.cgi', '*.conf'],},
-    entry_points={'console_scripts': ['gitserve = gitserve:main',],},
+    entry_points={'console_scripts': ['git-serve = gitserve:main',],},
     zip_safe=False,
     include_package_data = True,
 )
