diff --git i/configure.ac w/configure.ac
index 1ea2116..dc734ee 100644
--- i/configure.ac
+++ w/configure.ac
@@ -146,13 +146,6 @@ AM_CONDITIONAL([MANPAGE], [test "x$manpage" = "xyes"])
 # Debugging and unit tests
 #
 
-if test "x$debug" = "xyes"; then
-	CXXFLAGS="-O0 -g -DDEBUG"
-	CFLAGS="-O0 -g -DDEBUG"
-else
-	CXXFLAGS="-O2 -DNDEBUG"
-	CFLAGS="-O2 -DNDEBUG"
-fi
 AM_CONDITIONAL([DEBUG], [test "x$debug" = "xyes"])
 
 if test "x$testsuite" = "xifdebug" -a "x$debug" = "xyes"; then
