From: James Rowe <jnrowe@gmail.com>
Subject: Disable optimisation flags
Status: LOCAL

diff --git i/configure.ac w/configure.ac
index acbdc0f..87f2fba 100644
--- i/configure.ac
+++ w/configure.ac
@@ -146,13 +146,6 @@ AM_CONDITIONAL([MANPAGE], [test "x$manpage" = "xyes"])
 # Debugging and unit tests
 #
 
-if test "x$debug" = "xyes"; then
-	CXXFLAGS="${CXXFLAGS} -O0 -g -DDEBUG"
-	CFLAGS="${CFLAGS} -O0 -g -DDEBUG"
-else
-	CXXFLAGS="-O2 ${CXXFLAGS} -DNDEBUG"
-	CFLAGS="-O2 ${CFLAGS} -DNDEBUG"
-fi
 AM_CONDITIONAL([DEBUG], [test "x$debug" = "xyes"])
 
 if test "x$testsuite" = "xifdebug" -a "x$debug" = "xyes"; then
