diff --git i/setup.py w/setup.py
index cd561aa..74e3bdb 100644
--- i/setup.py
+++ w/setup.py
@@ -28,6 +28,9 @@ For more information, docs and many examples please checkout the `home page`_:
 
 def main():
     install_requires=['virtualenv>=1.6.1']
+    if sys.version_info[:2] < (2, 7):
+        install_requires.append('argparse')
+
     setup(
         name='tox',
         description='virtualenv-based automation of test activities',
@@ -40,7 +43,7 @@ def main():
         author_email='holger@merlinux.eu',
         packages=['tox', ],
         entry_points={'console_scripts': 'tox=tox:cmdline'},
-        install_requires=install_requires+['argparse', 'py>=1.4.0a2', ],
+        install_requires=install_requires+['py>=1.4.0a2', ],
         zip_safe=True,
         classifiers=[
             'Development Status :: 5 - Production/Stable',
