From: James Rowe <jnrowe@gmail.com>
Subject: Build fixes
Status: LOCAL

diff --git i/Makefile w/Makefile
index d064ab4..c83bfba 100644
--- i/Makefile
+++ w/Makefile
@@ -2,17 +2,17 @@ PACKAGE = yalias
 VERSION = 0.7
 DISTNAME = $(PACKAGE)-$(VERSION)
 
-prefix = /usr/local
+prefix = $(DESTDIR)/usr
 bindir = $(prefix)/bin
 sharedir = $(prefix)/share
 
 YALIASDIR = $(sharedir)/yalias
 SYSTEM_CONFIG = $(YALIASDIR)/system.yaliasrc
 
-CFLAGS = `pkg-config --cflags gtk+-2.0` \
+CFLAGS += `pkg-config --cflags gtk+-2.0` \
 	-DSYSTEM_CONFIG=\"$(SYSTEM_CONFIG)\" \
 	-DVERSION=\"$(VERSION)\"
-LDFLAGS = `pkg-config --libs gtk+-2.0` -lX11
+LDFLAGS += `pkg-config --libs gtk+-2.0` -lX11
 
 yalias: yalias.c
 	$(CC) $(CFLAGS) -o yalias yalias.c $(LDFLAGS)
