diff --git i/setup.py w/setup.py
index 331cb21..c4e16c8 100644
--- i/setup.py
+++ w/setup.py
@@ -16,7 +16,7 @@ setup(
     license = 'GNU LGPL',
     py_modules = ['freshen'],
     packages = ['freshen'],
-    data_files = [('freshen', ['freshen/languages.yml'])],
+    package_data = {'freshen': ['languages.yml']},
     entry_points = {
         'nose.plugins.0.10': [
             'freshen = freshen:FreshenNosePlugin',
