--- wingide/wing.orig	2006-07-29 02:00:08.000000000 +0100
+++ wingide/wing	2009-08-21 14:39:51.000000000 +0100
@@ -1,3 +1,15 @@
 #!/bin/sh
+# Force system GTK from first run
+if [ ! -f ~/.wingide3/use-system-gtk ]; then
+  mkdir -p ~/.wingide3
+  touch ~/.wingide3/use-system-gtk
+  python -c 'from ConfigParser import ConfigParser as CP
+from os.path import expanduser as eu
+c = CP(); c.read(eu("~/.wingide3/preferences"))
+c.has_section("user-preferences") or c.add_section("user-preferences")
+c.set("user-preferences", "gui.use-system-gtk", "true")
+c.write(open(eu("~/.wingide3/preferences"), "w"))'
+fi
+
 WINGHOME="/opt/wingide"; export WINGHOME
 . "${WINGHOME}/run-wing.sh"
