From: James Rowe <jnrowe@gmail.com>
Subject: Non-executable shell scripts belong in /usr/share.
Status: LOCAL

diff --git a/setup.py b/setup.py
index ca51079..de550fa 100644
--- a/setup.py
+++ b/setup.py
@@ -137,8 +137,7 @@ setup(
 
     platforms = ['Any'],
 
-    scripts = ['virtualenvwrapper.sh', 
-               ],
+    scripts = [],
 
     provides=['virtualenvwrapper',
               'virtualenvwrapper.user_scripts',
@@ -155,6 +154,9 @@ setup(
                                      package=PROJECT,
                                      only_in_packages=False,
                                      ),
+    data_files = [
+	("share/virtualenvwrapper", ["virtualenvwrapper.sh", ]),
+    ],
 
     entry_points = {
         #'console_scripts': [ 'venvw_hook = virtualenvwrapper.hook_loader:main' ],
