From: James Rowe <jnrowe@gmail.com>
Subject: Use the standard library version of simplejson, if available.
Status: LOCAL

diff --git a/setup.py b/setup.py
index e7f10ec..70c6208 100644
--- a/setup.py
+++ b/setup.py
@@ -13,6 +13,11 @@ except:
 sys.path.insert(0, os.path.join(cur_dir, 'src'))
 from github.version import get_version
 
+pkg_requires = ['setuptools', ]
+# simplejson is included in since Python 2.6 as json.
+if sys.version_info[0] < 3 and sys.version_info[1] < 6:
+    pkg_requires.append('simplejson')
+
 setup(
     name = "github-cli",
     version = get_version('short'),
@@ -24,7 +29,7 @@ setup(
     author_email = 'jhmsmits@gmail.com',
     packages = find_packages('src'),
     package_dir = {'': 'src'},
-    install_requires = ['setuptools', 'simplejson'],
+    install_requires = pkg_requires,
     entry_points="""
     [console_scripts]
     ghi = github.issues:main
diff --git a/src/github/issues.py b/src/github/issues.py
index 46a3a35..cb82a49 100644
--- a/src/github/issues.py
+++ b/src/github/issues.py
@@ -5,10 +5,13 @@ import webbrowser as browser
 from optparse import OptionParser
 
 try:
-    import simplejson
+    import json
 except ImportError:
-    print "error: simplejson required"
-    sys.exit(1)
+    try:
+        import simplejson as json
+    except ImportError:
+        print "error: simplejson required"
+        sys.exit(1)
 
 from github.utils import urlopen2, get_remote_info, edit_text, \
     get_remote_info_from_option, get_prog, Pager, wrap_text, get_underline
@@ -306,7 +309,7 @@ class Commands(object):
         args_list.insert(0, base_url)
         url = "/".join(args_list)
         page = urlopen2(url, **kwargs)
-        result = simplejson.load(page)
+        result = json.load(page)
         page.close()
         if result.get('error'):
             handle_error(result)
