From: James Rowe <jnrowe@gmail.com>
Subject: Correctly installs only poster package
Status: LOCAL

diff --git i/setup.py w/setup.py
index eb6775a..312529b 100644
--- i/setup.py
+++ w/setup.py
@@ -1,4 +1,4 @@
-from setuptools import setup, find_packages
+from setuptools import setup
 import poster
 
 # To update version number, edit:
@@ -29,7 +29,7 @@ multipart/form-data encoding of string or file parameters""",
       url='http://atlee.ca/software/poster',
       download_url='http://atlee.ca/software/poster/dist/%s' % version,
       license='MIT',
-      packages=find_packages(exclude='tests'),
+      packages=['poster', ],
       include_package_data=True,
       zip_safe=True,
       extras_require = {'poster': ["buildutils", "sphinx"]},
