From: James Rowe <jnrowe@gmail.com>
Subject: Path fix for multi-ABI support (Gentoo only)
Status: REJECTED

diff --git i/tests/_setup.py w/tests/_setup.py
index c410274..a23ca5b 100644
--- i/tests/_setup.py
+++ w/tests/_setup.py
@@ -3,4 +3,4 @@ import sys
 # Forcibly insert path for `setup.py build` output, so that we import from the
 # ``2to3`` converted sources.  This is an ugly hack, but it saves an enormous
 # amount of grief in handling Python 2 and 3.
-sys.path.insert(0, 'build/lib')
+sys.path.insert(0, 'build-%s/lib' % ".".join(map(str, sys.version_info[:2])))
