From: James Rowe <jnrowe@gmail.com>
Subject: Remove unused requests dependency
Status: LOCAL

diff --git i/setup.py w/setup.py
index 67ff2d6..e9be0b6 100644
--- i/setup.py
+++ w/setup.py
@@ -13,7 +13,6 @@ setup(
     include_package_data=True,
     install_requires=[
 #        'blessings',
-        'requests',
     ],
     long_description=(open('README.rst').read() + open(
         os.path.join('docs', 'HISTORY.txt')).read()),
