From: James Rowe <jnrowe@gmail.com>
Subject: Use open() for Python 3 compatibility.
Status: LOCAL

diff --git a/setup.py b/setup.py
index 5ce0829..a8354c1 100644
--- a/setup.py
+++ b/setup.py
@@ -17,7 +17,7 @@ from setuptools import setup, find_packages
 #=========================================================
 #inspection
 #=========================================================
-vh = file(os.path.join(lib_path, "cloud_sptheme", "__init__.py"))
+vh = open(os.path.join(lib_path, "cloud_sptheme", "__init__.py"))
 VERSION = re.search(r'^__version__\s*=\s*"(.*?)"\s*$', vh.read(), re.M).group(1)
 vh.close()
 #=========================================================
@@ -56,7 +56,8 @@ at `<http://packages.python.org/cloud_sptheme>`_.
         'Programming Language :: Python',
         'Topic :: Documentation',
         'Topic :: Software Development :: Documentation',
-    ]
+    ],
+    use_2to3 = True,
 )
 #=========================================================
 #EOF
