diff --git i/Makefile w/Makefile
index e77c55c..09893d9 100644
--- i/Makefile
+++ w/Makefile
@@ -5,12 +5,13 @@ CC        = cc
 
 CFLAGS    += -DSHAPE -lXext # comment out this line if you haven't got the x extentions library
 #CFLAGS    += -DDEBUG -ggdb3 # -DDEBUG_EVENTS -DSYNC # for debugging
-CFLAGS    += -lX11 -I$(XROOT)/include -L$(XROOT)/lib
+#CFLAGS    += -lX11 -I$(XROOT)/include -L$(XROOT)/lib
+LDFLAGS   += -lX11
 
 all: matwm2
 
 matwm2:
-	$(CC) $(CFLAGS) *.c -o $@
+	$(CC) $(CFLAGS) $(LDFLAGS) *.c -o $@
 
 clean:
 	rm -f matwm2
