diff --git i/Makefile w/Makefile
index 0a2f738..3c93782 100644
--- i/Makefile
+++ w/Makefile
@@ -6,8 +6,8 @@ RM        = rm -f
 
 CFLAGS    += -DSHAPE # comment out this line to disable shape support
 LDFLAGS   += -lXext # if shape support is disabled, this normally is no longer necessary and schould probally be commented out too
-CFLAGS    += -I$(XROOT)/include
-LDFLAGS   += -lX11 -L$(XROOT)/lib
+CFLAGS    +=
+LDFLAGS   += -lX11
 #CFLAGS    += -DDEBUG -ggdb3 # -DDEBUG_EVENTS -DSYNC # for debugging
 #CFLAGS    += -pedantic -std=c89 -Wall -W -Wno-unused-parameter
 CFLAGS    += -DVERSION=\"0.0.92\"
@@ -19,7 +19,7 @@ HEADERS   = all.h defcfg.h ewmh.h matwm.h mwm_hints.h
 all: matwm2
 
 matwm2: $(OBJFILES) $(HEADERS)
-	$(CC) $(LDFLAGS) $(OBJFILES) -o $@
+	$(CC) $(OBJFILES) -o $@ $(LDFLAGS)
 
 clean:
 	$(RM) matwm2 *.o
